### ISSUE TITLE
Should tests always be executed in incognito/private browsing mode?

### ISSUE THREAD

**Issue by jscholes** [w3c/aria-at#368]
For some test plans, we instruct users to execute a command which will have a different impact based on their browser state.  The primary example being a screen reader command to jump to the next unvisited link, which will fail if the link's destination is present in the browser history.

Consider that in the same test plan, we are likely to also instruct the tester to activate the link at some point.  If/when they revisit the test page, e.g. to carry out tests with a different AT in the same browser, the command may not work as expected.

A possible concern here is that testers must be signed into GitHub to assign themselves to test plans or carry out testing, and they'll need to reauthenticate each time they open a private tab.  But this should only need to be done once per testing session.

**Comment by mcking65** [comment]
Are there any scenarios other than visited/unvisited link? Do we need to use the command that depends on link visitation state? Why not just next link?

**Comment by jscholes** [comment]
@mcking65 

> Do we need to use the command that depends on link visitation state? Why not just next link?

I don't think there is a next link command in JAWS anymore, only visited or unvisited.  The help file lists Tab and Shift+Tab for next/previous link though, which seems misleading at best.  So let me know if the command does exist; happy to switch to it exclusively if so.

> Are there any scenarios other than visited/unvisited link?

None have come up so far.
